### PR TITLE
Remove old model queries

### LINKED ISSUE #1558: Renaming retains data for original model
Renaming from a model and creating a new model with the old name causes to retain old data like profiling data and tables.

### PR DESCRIPTION
**Pull request by djbarnwal**
Closes #1558

### CODE REVIEW

**Comment by hamilton** [comment]
@AdityaHegde @djbarnwal not sure if this PR fixes it, but another place to add invalidation is when you delete a model. I'm seeing on other branches that if I delete a model, then create a new one and name it the same thing, the old data persists.

File: web-local/src/lib/svelte-query/invalidation.ts [hunk 1]
```diff
@@ -90,3 +94,28 @@ export function invalidateProfilingQueries(
...
+    queryClient.removeQueries(
+      getRuntimeServiceGetFileQueryKey(instanceId, path)
+    ),
+    queryClient.removeQueries(
+      getRuntimeServiceGetCatalogEntryQueryKey(instanceId, name)
+    ),
+  ]);
+
+  // remove profiling queries
+  return queryClient.removeQueries({
```
Comment (AdityaHegde) [inline web-local/src/lib/svelte-query/invalidation.ts:116]: We should run this for metrics as well. Do something similar to `getInvalidationsForPath` from `invalidations.ts`

File: web-local/src/lib/svelte-query/actions.ts [hunk 2]
```diff
@@ -100,6 +104,9 @@ export async function deleteFileArtifact(
       notifications.send({ message: `Deleted ${getLabel(type)} ${name}` });
     }
 
+    if (type == EntityType.Model)
```
Comment (AdityaHegde) [inline web-local/src/lib/svelte-query/actions.ts:107]: Remove this check once we add support for metrics as well.

File: web-local/src/lib/svelte-query/invalidation.ts [hunk 3]
```diff
@@ -73,6 +73,14 @@ export const invalidateMetricsViewData = (
   });
 };
 
+export function invalidationForMetricsViewData(query, metricsViewName: string) {
+  const r = new RegExp(
```
Comment (AdityaHegde) [inline web-local/src/lib/svelte-query/invalidation.ts:77]: We should have a single method for matching metrics view query. Perhaps refactor invalidateMetricsViewData to call this method as well?

**Review by AdityaHegde — APPROVED** [review]

**Pull request merged**